README.md

```diff
@@ -39,7 +39,7 @@ to install and begin creating APKs and AABs.
 
 (for the develop branch: `pip install git+https://github.com/kivy/python-for-android.git`)
 
-Test that the install works with:
+Test that theinstall works with:
```

Comment:
oops

Reply:
Ouch!
For some reason I've read: "Test that install works with:" 🫣

@Julian-O can you take care of it?

Reply:
I just spent some time trying to figure out how I managed to break this comment.

1. Fix a legitimate typo in a comment of no consequence.
2. Accidentally delete a space in another file.
3. A few days later, at commit time, see that delta for the lost space, but don't notice it is the wrong way. Think "Ah, yeah, the typo I fixed. I will include that in this PR."

Sorry. PR with both inconsequential fixes coming very soon.